src/sttn/eval/analyst_eval.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
Could you please move the logic to Python modules & classes?
Notebooks are not the best interface to manage Python code. Ideally, the runner notebook should contain only a few lines of code to start the evaluation.

Reply:
Done! The new version has all evaluators in the separate file